docs/dev_guide/term-mapping-quick-ref.rst

```diff
@@ -0,0 +1,164 @@
+CUDA* to SYCL* Term Mapping Quick Reference
+===========================================
+
+This quick reference maps common CUDA* terms to SYCL* terms, to help start the
+migration process.
+
+
+Architecture Terminology Mapping
+--------------------------------
+
++-------------------------------+------------------------------------+-------------------------------------------+
+| CUDA Capable GPU              + SYCL Capable GPU from Intel                                                    |
++===============================+====================================+===========================================+
+|                               | **Xe-LP and prior generations**    | **Xe-HPG and Xe-HPC**                     |
++-------------------------------+------------------------------------+-------------------------------------------+
+| CUDA Core                     | Execution Unit (EU)                | Vector Engine & Matrix Engine (XVE & XMX) |
++-------------------------------+------------------------------------+-------------------------------------------+
+| Streaming Multiprocessor (SM) | Subslice(SS) or Dual Subslice(DSS) | Xe-Core                                   |
++-------------------------------+------------------------------------+-------------------------------------------+
+| Processor Clusters (PC)       | Slice                              | Xe-Slice                                  |
++-------------------------------+------------------------------------+-------------------------------------------+
+| N/A                           | N/A                                | Xe-Stack                                  |
++-------------------------------+------------------------------------+-------------------------------------------+
+
+Execution Model Mapping
+-----------------------
+
+.. list-table::
+   :widths: 50 50
+   :header-rows: 1
+
+   * - CUDA
+     - SYCL
+   * - Thread
+     - Work-item
+   * - Warp
+     - Sub-group
+   * - Block
+     - Work-group
+   * - Grid
+     - ND-range
+
```

Comment:
should change the upper case to lower case

Reply:
I'm following the MS style guide (https://learn.microsoft.com/en-us/style-guide/scannable-content/tables): Since this table lists comparable concepts (not programming key words) I'm treating the table cell entries in sentence case. SYCL docs do similar (https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#_work_group_data_parallel_kernels). Glad to discuss!

Reply:
OK